cernopendata/modules/fixtures/data/records/cms-simulated-datasets-Run1-datascience.json

```diff
@@ -106,8 +106,8 @@
           "configuration_files": [
             {
               "process": "SIM",
-              "title": "Configuration file for SIM step step0_ttbar.py",
-              "url": "https://github.com/cms-opendata-analyses/TrackerRecHitProducerTool/blob/v12.0.0/configs/step0_ttbar.py"
+              "title": "Configuration file for SIM",
```

Comment:
It is enough to have simply "Configuration file" here, as the step will be taken from `process` field.  I'll amend this while merging...